core/src/tangram.h

```diff
@@ -154,6 +156,7 @@ bool getDebugFlag(DebugFlags _flag);
 void toggleDebugFlag(DebugFlags _flag);
 
 void runOnMainLoop(std::function<void()> _task);
+void runAsyncTask(std::function<void()> _task);
```

Comment:
True, it will also have to be re-factored when we have multiple instancing, specifically around which instance's update/render needs to be performed. But thats a job of another PR :D.

Will document it here.